alf/algorithms/rl_algorithm_test.py

```diff
@@ -124,28 +132,66 @@ def step(self, action):
             reward=self._rewards[action],
             discount=torch.zeros(self._batch_size),
             prev_action=self._prev_action,
-            env_id=torch.arange(self._batch_size))
-
+            env_id=torch.arange(self._batch_size, dtype=torch.int32))
         self._prev_action = action
         return self._current_time_step
 
     def current_time_step(self):
         return self._current_time_step
 
 
-class Config(object):
-    def __init__(self):
-        self.unroll_length = 5
+class RLAlgorithmTest(unittest.TestCase):
+    def test_on_policy_algorithm(self):
+        config = TrainerConfig(
+            root_dir='/tmp/rl_algorithm_test', unroll_length=5, num_envs=1)
```

Comment:
Might be possible to update this test case with ```tempfile``` as in ```_test_off_policy_algorithm```.

Reply:
I put the comment at the wrong line 177:
        # root_dir is not used. We have to give it a value because
        # it is a required argument of TrainerConfig.

Reply:
I put the comment at the wrong line 177:
        # root_dir is not used. We have to give it a value because
        # it is a required argument of TrainerConfig.